src/main/java/edu/harvard/iq/dataverse/workflow/step/Success.java

```diff
@@ -0,0 +1,47 @@
+package edu.harvard.iq.dataverse.workflow.step;
+
+/**
+ * Result returned when step execution succeeds.
+ */
+public class Success implements WorkflowStepResult {
+    
+    private final String reason;
+    private final String message;
+    
+    public Success( String reason ) {
+        this(reason, null);
+    }
+    
+    /**
+     * Constructs a new success message.
+     * @param reason Technical reason (for logs etc.).
+     * @param message Human readable reason.
+     */
+    public Success(String reason, String message) {
+        this.reason = reason;
+        this.message = message;
+    }
+    
+    /**
+     * Holds the technical reason for the success, useful for debugging the problem.
+     * @return the technical reason for the problem.
+     */
+    public String getReason() {
+        return reason;
+    }
+
+    /**
+     * Holds the user-friendly message explaining the failure.
```

Comment:
Not very "friendly", to claim it's a failure, even though it's a success! :)
(I know, it's just a copy-and-paste thing...)

Reply:
"Sadly this workflow successfully completed as designed. Please contact support if you are disappointed because you were expecting something less reliable." :-) - fixed.